apps/client-server/src/app/post/services/post-file-resizer/post-file-resizer.service.ts

```diff
@@ -181,20 +182,16 @@ export class PostFileResizerService {
     let counter = 0;
     while (await this.isFileTooLarge(s, maxBytes)) {
       counter += 1;
-      const resizePercent = counter * 0.05;
-      if (resizePercent >= 1) {
-        break;
-      }
+      const resizePercent = 1 - counter * 0.05;
+      if (resizePercent <= 0.1) break;
```

Comment:
There was a broken logic. It was trying to scale image to 0.05, then 0.025 etc. Now it tries to do 0.95 then 0.90 correctly.

Reply:
praise: good catch